### PR TITLE
fix typo

### LINKED ISSUE #811: typo in template file CONTRIBUTING.md
## Description of the bug

The template file `CONTRIBUTING.md` has a typing error:

```
Add any new flags/options to nextflow_schema.json with help text (with nf-core schema build .)
```

## Additional context
A PR is on the way.

### PR DESCRIPTION
**Pull request by subwaystation**
## PR checklist

 - [x] This comment contains a description of changes (with reason)
 - [ ] `CHANGELOG.md` is updated
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Documentation in `docs` is updated

This PR fixes #811

### CODE REVIEW

**Comment by jfy133** [comment]
I put the `.` as it makes it a complete command because python _requires_ that the location is specified. Maybe with replace  `<PIPELINE PATH>` or similar instead

**Comment by subwaystation** [comment]
Oh. I didn't realize that. So maybe there is only a `.` missing at the end of the sentence? I am fine with the `.` as you had it before. I just thought it is a typo. Sorry.

**Comment by jfy133** [comment]
Correct, missing full-stop at the end of the sentence as well!

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.github/CONTRIBUTING.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ If you wish to contribute a new step, please use the following coding standards:
 2. Write the process block (see below).
 3. Define the output channel if needed (see below).
 4. Add any new flags/options to `nextflow.config` with a default (see below).
-5. Add any new flags/options to `nextflow_schema.json` with help text (with `nf-core schema build .`)
+5. Add any new flags/options to `nextflow_schema.json` with help text (with `nf-core schema build`).
```
Comment (apeltzer) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.github/CONTRIBUTING.md:72]:
```suggestion
5. Add any new flags/options to `nextflow_schema.json` with help text (with `nf-core schema build .`).
```

**Review by apeltzer — CHANGES_REQUESTED** [review]

**Review by jfy133 — APPROVED** [review]

**Pull request merged**